src/main/resources/db/changelog/changesets/changelog_20231221T165233Z.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="harperjon (generated)" id="1703177560263-1">
```

Comment:
I had added [a comment](https://github.com/gridsuite/dynamic-simulation-server/pull/66/files#diff-b4ebb53fb33ae582b56ccbea3d0a7c7730b53c23664ba77d4522b5ba997db025R25) to explain why we do it in
```suggestion
    <changeSet author="harperjon (generated)" id="1703177560263-1">
        <comment>Missing update from #37</comment>
```